io/module_write_netcdf_parallel.F90

```diff
@@ -171,7 +172,7 @@ subroutine write_netcdf_parallel(fieldbundle, wrtfb, filename, mpi_comm, mype, i
             ncerr = nf90_def_var(ncid, trim(fldName), NF90_FLOAT, &
                     (/im_dimid,jm_dimid,pfull_dimid,time_dimid/), varids(i), &
                     shuffle=.false.,deflate_level=ideflate,&
-                    chunksizes=(/im,jm,1,1/),cache_size=40*im*jm); NC_ERR_STOP(ncerr)
+                    chunksizes=(/im,jm,1,1/)); NC_ERR_STOP(ncerr)
```

Comment:
Must two of the chunksizes be 1?

Reply:
No, but this seems to be the most common access pattern (reading a 2d horizontal slice).

Reply:
Now changed so that chunksize can be specified at run time.  Default is to use MPI decompisition size..